core/kernel/pathname_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../../spec_helper'
+require 'pathname'
```

Comment:
`pathname` is a standard library, could you move the spec to `library/pathname/Pathname_spec.rb`?
The reason is no standard libraries should be `require`-d when running `core/` specs.